adding-a-new-service.md

```diff
@@ -0,0 +1,15 @@
+# How to add a new service
```

Comment:
imho this should be parts of our docs, not sure if it is needed here additionally

Reply:
maybe put it in the docs and put a link in the README?